ocaml/xapi-cli-server/records.ml

```diff
@@ -4976,3 +4976,37 @@ let repository_record rpc session_id repository =
           ()
       ]
   }
+
+let vtpm_record rpc session_id vtpm =
+  let _ref = ref vtpm in
+  let empty_record =
+    ToGet (fun () -> Client.VTPM.get_record rpc session_id !_ref)
+  in
+  let record = ref empty_record in
+  let x () = lzy_get record in
```

Comment:
I wasn't aware of this lazy business and this code is repeated over and over in this module. I wonder why the OCaml Lazy module could not have been used for this. Now it's too late because we have this record but it looks convoluted.

Reply:
This module also has hundreds of calls to functions without labelling their arguments, it's in the best status regarding maintenance.

Last time I did a cleanup here around inconsistent display of lists xenrt broke so I'm a bit wary of being too aggressive here